ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/event-handlers.md

```diff
@@ -35,7 +46,32 @@
         delay: <int>
 
         # Configure event trigger condition.
-        trigger: <str; "on-boot" | "on-logging" | "on-startup-config">
+        trigger: <str; "on-boot" | "on-logging" | "on-startup-config" | "on-maintenance">
+
+        # Settings required for trigger 'on-maintenance'.
+        trigger_on_maintenance:
+          operation: <str; "enter" | "exit"; required>
+
+          # Trigger condition occurs on maintenance operation of specified BGP peer.
+          bgp:
+            neighbor_address_ipv4: <str>
+            neighbor_address_ipv6: <str>
+            peer_group_name: <str>
```

Comment:
do we really need three keys, i think it should be one. Based on user they can add IPv4/IPv6 or peer group name.

Reply:
Done